torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -400,8 +404,22 @@ def is_zero_dim(meta):
         # Other ops
         ############################################################
         if target == torch.ops.aten._adaptive_avg_pool2d.default:
-            # assumes output size is (1, 1)
-            return self.call_function_prop_meta(ttnn.global_avg_pool2d, (args[0],), kwargs)
+            output_size = None
+            arg0_size = None
+            if len(args) > 1:
+                output_size = args[1]
+            elif "output_size" in kwargs:
+                output_size = kwargs["output_size"]
+            new_graph_node_meta = self.get_new_graph_node_meta()
+            if hasattr(args[0], "node") and args[0].node.name in new_graph_node_meta:
+                arg0_meta = new_graph_node_meta[args[0].node.name]
+                arg0_size = list(arg0_meta["val"].size())
+            # special case: if output_size is same with input last two dimensions
+            # then actually this op do nothing
+            if output_size is not None and arg0_size is not None and arg0_size[-2:] == output_size:
+                return args[0]
+            # TODO: no ttnn op can convert _adaptive_avg_pool2d
```

Comment:
`ttnn.global_avg_pool2d` actually can handle the case of output_size `(1, 1)` I guess

But I'm fine to convert it now (which is not used) or later with the actual `ttnn.avg_pool2d` implemented